src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,22 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+import './CommentInfo.scss';
+
+export const CommentInfo = ({ comment }) => (
+  <div className="CommentInfo">
```

Comment:
it's a good practice to use object destructing. You can use destructing inside accepted props ` ({ comment }) ` or inside a component body like below:
```suggestion
export const CommentInfo = ({ comment }) => {
  const {...} = comment;
  
  return (
    <div className="CommentInfo">
```